### DISCUSSION TITLE
Arduino GFX rgb interface

### DISCUSSION THREAD

**Discussion by deptou31** [moononournation/Arduino_GFX#297] [Q&A]
hello, first of all thank you for this great work, I mounted a board based on the ESP32-S3 and it works wonderfully with a 4.3" or 5" RGB 565 screen.
Here is my problem, I tried to switch to an ESP32-S2 but there I encounter a lot of problems with the compilation... have you tried this type of module with RGB LCDs?

thank you for your feedback

**Comment by moononournation** [comment]
only ESP32-S3 support RGB display

**Reply by deptou31** [reply]
Hi, thanks for reply.
